include/libdev/machlog/oreholo.hpp

```diff
@@ -57,6 +57,8 @@ class MachLogOreHolograph : public MachActor
 	
 	friend class MachLogMineralSite;
 
+	void assignToDifferentRace( MachLogRace& newRace );
```

Comment:
No `override` because it triggers zillion more warnings because other methods are not marked override.